{"name": "test_4", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 4", "trace": "def test_4():\n        print(\"test_4\")\n>       assert 1 == 4\nE       assert 1 == 4\n\n..\\test_case\\test_case_02.py:3: AssertionError"}, "attachments": [{"name": "stdout", "source": "cf57c6aa-a901-4cef-ab02-94a6ecf3682a-attachment.txt", "type": "text/plain"}], "start": 1607071028030, "stop": 1607071028031, "uuid": "2e9451e5-af59-40e4-8d2d-354cd23322c3", "historyId": "2f847e93cec3be246e1f2e9c40e99306", "testCaseId": "3981143484eba921566f524e5115510d", "fullName": "test_case.test_case_02#test_4", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_02"}, {"name": "host", "value": "LAPTOP-UDME04BU"}, {"name": "thread", "value": "25160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_02"}]}